packages/core/src/types/interfaces/i-cell-data.ts

```diff
@@ -48,6 +49,7 @@ export interface ICellData {
 export interface ICellDataForSheetInterceptor extends ICellData {
     interceptorStyle?: Nullable<IStyleData>;
     isInArrayFormulaRange?: Nullable<boolean>;
+    customRender?: Nullable<ICellCustomRender[]>;
```

Comment:
We should provide a similar interceptor on the rendering package instead of making into core.